producer-c/producer-cloudwatch-integ/KvsProducerSampleCloudwatch.cpp

```diff
@@ -180,26 +197,53 @@ INT32 main(INT32 argc, CHAR *argv[])
             frame.presentationTs = frame.decodingTs;
             frameIndex++;
         }
+        if (STATUS_FAILED(retStatus)) {
+            DLOGE("Failed with status 0x%08x\n", retStatus);
+         }
```

Comment:
Minor: You could do CHK_LOG_ERR

Reply:
I am not doing CHK_LOG_ERR because I do not want to jump to Cleanup after this. I want it to be logged in logged in cloudwatch

Reply:
Oh never mind. I was wrong. Let me change it.

Reply:
Done